### PR TITLE
Kubernetes v0.19.0

### PR DESCRIPTION
**Pull request by otaviof**
Upgrading Go modules and replacement statements to use `v0.19.0-rc.3`.

### CODE REVIEW

**Comment by gabemontero** [comment]
the org gets rather precise when it comes to the exact commit message @otaviof for these gomod/vendor changes

take a look at the commit message from either my samples or @coreydaley 's OCM PRs and try to follow that pattern

File: go.mod [hunk 1]
```diff
@@ -22,15 +21,15 @@ require (
...
-	k8s.io/component-base v0.18.0
+	golang.org/x/sys v0.0.0-20200622214017-ed371f2e16b4
+	k8s.io/api v0.19.0-rc.3
+	k8s.io/apimachinery v0.19.0-rc.3
+	k8s.io/client-go v0.19.0-rc.3
+	k8s.io/component-base v0.19.0-rc.3
 	k8s.io/klog v1.0.0
-	k8s.io/kubectl v0.18.0
+	k8s.io/kubectl v0.19.0-rc.3
 	k8s.io/kubernetes v1.18.0
```
Comment (adambkaplan) [inline go.mod:33]: `k8s.io/kubernetes` also needs to be bumped, version tag is `v1.19.0-rc.3`. This tends to be the most difficult one.

Comment (otaviof) [reply go.mod]: Indeed, takes the most effort in this part. I've updated that dependency in the last interaction, please consider.

File: go.mod [hunk 2]
```diff
@@ -12,56 +12,58 @@ require (
...
 )
 
 replace (
 	github.com/containerd/containerd => github.com/containerd/containerd v1.3.2
 	github.com/docker/docker => github.com/openshift/moby-moby v1.4.2-0.20190308215630-da810a85109d
 	github.com/docker/libnetwork => github.com/docker/libnetwork v0.8.0-dev.2.0.20171107005402-dcf79f8c7fcb
 	github.com/godbus/dbus => github.com/godbus/dbus v0.0.0-20181101234600-2ff6f7ffd60f
+	// testing changes
+	github.com/opencontainers/runc => github.com/opencontainers/runc v1.0.0-rc90
+	github.com/opencontainers/runtime-spec => github.com/opencontainers/runtime-spec v1.0.1
```
Comment (adambkaplan) [inline go.mod:43]: Is this replace necessary?

Comment (otaviof) [reply go.mod]: I'm removing this replace block, and running `go mod tidy`. As you can see it skewed `runc` and `runtime-spec` a bit. Should we keep with a replace statement instead?

Comment (otaviof) [reply go.mod:43]: I end up adding the replace statements again, due the errors we can see here: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_builder/167/pull-ci-openshift-builder-master-unit/1292732924029833216/build-log.txt

**Comment by adambkaplan** [comment]
/retest

The failing and flaking tests are not related to builds, and `e2e-aws-builds` and `e2e-aws-image-ecosystem` suites have both passed. I'm pretty confident we can drop WIP on this PR.

**Pull request merged**